tests/models/causalformer/test_encoder.py

```
@@ -0,0 +1,82 @@
+        embedding_size=embedding_size,
+        feature_dimensionality=feature_dimensionality,
+        ffn_hidden_dimensionality=ffn_hidden_dimensionality,
+        tau=tau,
+        dropout=dropout,
+    )
+
+    # Forward pass
+    try:
+        encoder(x)
+    except Exception as e:
+        assert False, f"Encoder forward pass failed with exception: {e}"
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)